isolated-contexts.bs

```diff
@@ -373,14 +373,12 @@ the associated [=environment settings object/global object=].
 
 ### Integrity ### {#html-integrity}
 
-A [=browsing context group=] has an <dfn for="browsing context group" export>
-integrity origin</dfn>, which is an [=origin=] or `null`.
+An <dfn export>integrity verification algorithm</dfn> is an
+[=implementation-defined=] algorithm that accepts a [=request=] and a
+[=response=], and returns a [=boolean=].
 
-A [=browsing context group=] has an <dfn for="browsing context group" export>
-integrity verification algorithm</dfn>, which is `null` or a [=user agent=]
-defined algorithm that accepts a [=request=] and a [=response=], and returns a
-[=boolean=]. A [=browsing context group=]'s [=integrity verification algorithm=]
-MUST be non-null if its [=integrity origin=] is non-null.
+A [=user agent=] holds an <dfn export>origin integrity verification map</dfn>,
+which is a [=map=] of [=origins=] to [=integrity verification algorithms=].
```

Comment:
How is this supposed to work for opaque origins? Are UAs supposed to supply independent algorithms for each (impossible... I guess), or are opaque origins just ignored for the purposes of this map. If the latter is the case, I would probably make this a map of [tuple origins](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-tuple) specifically?

Reply:
Good point, it should be tuple origins.